lib/autoyast.rb

```diff
@@ -262,26 +262,31 @@ def apply_url_extraction(xml)
     end
   end
 
-  def apply_extracted_files(scope)
-    return if !@system_description[scope] || !@system_description[scope].extracted
-
-    base = Pathname(@system_description.scope_file_store(scope).path)
-    Dir["#{base}/**/*"].sort.each do |path|
-      next if File.directory?(path)
-
-      relative_path = Pathname(path).relative_path_from(base).to_s
-      url = "`cat /tmp/description_url`/#{URI.escape(File.join(scope, relative_path))}"
-
-      @chroot_scripts << "mkdir -p '#{File.join("/mnt", File.dirname(relative_path))}'"
-      @chroot_scripts << "curl -o '#{File.join("/mnt", relative_path)}' \"#{url}\""
-    end
-
-    @system_description[scope].files.map do |file|
-      if file.user
-        @chroot_scripts << "chown #{file.user}:#{file.group} '#{File.join("/mnt", file.name)}'"
-      end
-      if file.mode
-        @chroot_scripts << "chmod #{file.mode} '#{File.join("/mnt", file.name)}'"
+  def apply_changed_files(scope)
```

Comment:
Assignment Branch Condition size for apply_changed_files is too high. [59.22/15]